includes/payments/class-edd-payment.php

```diff
@@ -1414,6 +1414,7 @@ public function modify_cart_item( $cart_index = false, $args = array() ) {
 		// Format the item_price correctly now
 		$merged_item['item_price']  = edd_sanitize_amount( $merged_item['item_price'] );
 		$new_subtotal               = floatval( $merged_item['item_price'] ) * $merged_item['quantity'];
+		$merged_item['tax']  = edd_sanitize_amount( $merged_item['tax'] );
 		$merged_item['price']       = edd_prices_include_tax() ? $new_subtotal : $new_subtotal + $merged_item['tax'];
```

Comment:
```suggestion
		$merged_item['item_price'] = edd_sanitize_amount( $merged_item['item_price'] );
		$new_subtotal              = floatval( $merged_item['item_price'] ) * $merged_item['quantity'];
		$merged_item['tax']        = edd_sanitize_amount( $merged_item['tax'] );
		$merged_item['price']      = edd_prices_include_tax() ? $new_subtotal : $new_subtotal + $merged_item['tax'];
```

Some spacing tweaks from https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#indentation

Reply:
@spencerfinnell, did you make this change or should I commit it?